### PR TITLE
[MINOR] docs(lakehouse-iceberg):  improvement lakehouse iceberg document

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
1. add "Manage Metadata Using Gravitino" link to lakehouse iceberg document
2. merge info blocks

### Why are the changes needed?
more user friendly

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
just docs

### CODE REVIEW

File: docs/lakehouse-iceberg-catalog.md [hunk 1]
```diff
@@ -58,7 +55,9 @@ You must download the corresponding JDBC driver to the `catalogs/lakehouse-icebe
 
 ### Catalog operations
 
-## Schema
+Please refer [Manage Metadata Using Gravitino](./manage-metadata-using-gravitino#catalogs-operations) for more details.
```
Comment (jerryshao) [inline docs/lakehouse-iceberg-catalog.md:58]: "refer to"

File: docs/lakehouse-iceberg-catalog.md [hunk 2]
```diff
@@ -70,7 +69,9 @@ You could put properties except `comment`.
 
 ### Schema operations
 
-## Table
+Please refer [Manage Metadata Using Gravitino](./manage-metadata-using-gravitino#schemas-operations) for more details.
```
Comment (jerryshao) [inline docs/lakehouse-iceberg-catalog.md:72]: "refer to"

File: docs/lakehouse-iceberg-catalog.md [hunk 3]
```diff
@@ -163,6 +161,8 @@ The Gravitino server doesn't allow passing the following reserved fields.
 
 ### Table operations
 
+Please refer [Manage Metadata Using Gravitino](./manage-metadata-using-gravitino#tables-operations) for more details.
```
Comment (jerryshao) [inline docs/lakehouse-iceberg-catalog.md:164]: ditto.

File: docs/lakehouse-iceberg-catalog.md (hunk 1 above)
Comment (FANNG1) [reply docs/lakehouse-iceberg-catalog.md]: done, and update hive and jdbc related docs to keep consistent

**Review by jerryshao — APPROVED** [review]

**Pull request merged**